src/main/resources/assets/js/application/interviewer/controllers/tdprInterviewerHomeController.js

```diff
@@ -80,12 +80,8 @@ define(['angular', 'application/interviewer/tdprInterviewerModule', 'application
         }
 
        $scope.showPreviousWeek = function() {
-            if($scope.hasNoteChanged || $scope.hasSlotChanged) {
-                Notification.warning({
-                    message: "You have changed your data. Submit or discard your changes!",
-                    delay: 2000});
-                return;
-            }
+            var isPossibleToChangeWeek = notifyAboutNotSubmittedChanges();
```

Comment:
This is a bit confusing - the function does a check but the name suggests it only notifies.